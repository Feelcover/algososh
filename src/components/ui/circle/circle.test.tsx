import TestRenderer from "react-test-renderer";
import { Circle } from "./circle";


describe("Корректная работа Circle", () => {
    it("Circle без буквы", () => {
        const circle = TestRenderer.create(<Circle/>).toJSON();
        expect(circle).toMatchSnapshot();
    });

    it("Circle c буквами", () => {
        const circle = TestRenderer.create(<Circle/>).toJSON();
        expect(circle).toMatchSnapshot();
    });
})