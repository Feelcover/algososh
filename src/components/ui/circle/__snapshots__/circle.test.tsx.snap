// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Корректная работа Circle Circle c буквами 1`] = `
<div
  className="content "
  data-testid="circle"
>
  <div
    className="text text_type_input text_color_input mb-4 absolute head element"
    data-testid="head"
  />
  <div
    className="circle   default"
    data-testid="circleDiv"
  >
    <p
      className="text text_type_circle text_color_input letter"
      data-testid="testCircle"
    />
  </div>
  <p
    className="text text_type_input text_color_input mt-4 absolute index"
  />
  <div
    className="text text_type_input text_color_input mt-4 absolute tail30 element"
    data-testid="tail"
  />
</div>
`;

exports[`Корректная работа Circle Circle без буквы 1`] = `
<div
  className="content "
  data-testid="circle"
>
  <div
    className="text text_type_input text_color_input mb-4 absolute head element"
    data-testid="head"
  />
  <div
    className="circle   default"
    data-testid="circleDiv"
  >
    <p
      className="text text_type_circle text_color_input letter"
      data-testid="testCircle"
    />
  </div>
  <p
    className="text text_type_input text_color_input mt-4 absolute index"
  />
  <div
    className="text text_type_input text_color_input mt-4 absolute tail30 element"
    data-testid="tail"
  />
</div>
`;
